### DISCUSSION TITLE
Appeal Ban on Discord

### DISCUSSION THREAD

**Discussion by bobgrey1997** [ldtteam/minecolonies#8560] [General]
A moderator banned me simply for providing my reason for not wanting to donate. Sure, it was in the wrong channel, but it was in response to someone in that channel suggesting I donate. No ill intent. They got fed up because the reason involved the unsupported versions policy. I never actually violated that policy in this case, just referred to it as a reason. I didn't actually break any rules in this case beyond simply being off-topic for the channel (as the entire discussion was at this point).
Why would anyone donate if this is the treatment?

I don't see anywhere to make an appeal, so here we are.

**Comment by marchermans** [comment]
The server administrators have reviewed your ban.
Your particular way of stating your unwillingness to contribute is fine. However, the way you expressed it violated several rules and the discord ToS by requesting channels that would be completely unmoderated as a consequence of your demand.

Your review as such has been denied.

**Reply by bobgrey1997** [reply]
What rules did it violate. I meant no violating by it.
Also, Discord does not specify a level of moderation. A simple filter bot will make it compliant.
Also, the suggestion about such a channel was part of a different discussion, for which I was already punished via an hour long mute.

**Reply by marchermans** [reply]
Sorry but we are not further discussing this. We reviewed the ban with new people which were not online and they still agreed.

The conversation will here by be locked.
